src/components/ui/Button.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+
+interface ButtonProps {
+  text: string; 
+  onClick?: () => void;
+}
+
+const Button: React.FC<ButtonProps> = ({ text, onClick }) => {
+  return (
+    <button onClick={onClick} style={styles.button}>
+      {text}
+    </button>
+  );
+};
+
+const styles = {
+  button: {
+    padding: '10px 20px',
+    borderRadius: '50px',
+    border: '2px solid #0000FF',
+    backgroundColor: 'transparent',
+    color: '#0000FF',
+    fontSize: '16px',
+    fontWeight: 'bold',
+    cursor: 'pointer',
+    transition: 'background-color 0.3s, color 0.3s',
+  },
+};
```

Comment:
It will be hard to reuse this component since you're limiting button to certain styling. Make use of variants and defaultVariant to make this reusable. Siguro, add an additional optional prop to pass in a variant, then set lang default variant.

What's our approach for variants? @shunindesuuu

Reply:
@arabellamejorada Kindly use cva for the variants!

use this for reference nalang thank you!
https://cva.style/docs/examples/react/tailwind-css